x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/evasion_suspicious_scrobj_load.json

```diff
@@ -0,0 +1,41 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies scrobj.dll loaded into unusual Microsoft processes. This usually means a malicious scriptlet is being executed in the target process.",
+  "from": "now-9m",
+  "index": [
+    "logs-endpoint.events.*",
+    "winlogbeat-*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Windows Suspicious Script Object Execution",
+  "query": "sequence by process.entity_id with maxspan=2m\n  [process where event.type in (\"start\", \"process_started\") and\n     /* process.code_signature.* fields need to be populated for 7.10 */\n     process.code_signature.subject_name == \"Microsoft Corporation\" and process.code_signature.trusted == true and\n     process.name not in (\"cscript.exe\",\n                          \"iexplore.exe\",\n                          \"MicrosoftEdge.exe\",\n                          \"msiexec.exe\",\n                          \"smartscreen.exe\",\n                          \"taskhostw.exe\",\n                          \"w3wp.exe\",\n                          \"wscript.exe\")]\n  [library where event.type == \"start\" and file.name == \"scrobj.dll\"]\n",
```

Comment:
Just commenting for posterity that we'll remove /* process.code_signature.* fields need to be populated for 7.10 */ in a follow-up PR.